AndroidManifest-common.xml

```diff
@@ -43,13 +43,16 @@
     <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES" />
     <uses-permission android:name="android.permission.EXPAND_STATUS_BAR" />
     <uses-permission android:name="android.permission.ACCESS_RESTRICTED_SETTINGS" />
-    
+
     <!-- for rotating surface by arbitrary degree -->
     <uses-permission android:name="android.permission.VIBRATE"/>
     <uses-permission android:name="android.permission.ROTATE_SURFACE_FLINGER" />
     <uses-permission android:name="android.permission.WAKEUP_SURFACE_FLINGER" />
     <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
 
+    <!-- Floating window -->
+    <uses-permission android:name="com.yaap.permission.START_FREEFORM" />
+
```

Comment:
please move this too in [here](https://github.com/LawnchairLauncher/lawnchair/blob/15-dev/lawnchair/AndroidManifest.xml) Thanks!